src/services/listCourtCases.ts

```diff
@@ -12,6 +11,16 @@ import filterByReasonAndResolutionStatus from "./filters/filterByReasonAndResolu
 import courtCasesByOrganisationUnitQuery from "./queries/courtCasesByOrganisationUnitQuery"
 import leftJoinAndSelectTriggersQuery from "./queries/leftJoinAndSelectTriggersQuery"
 
+function formatName(name: string) {
+  let splitName = name.replace(/\*|\s+/g, "%")
+
+  if (!splitName.endsWith("%")) {
+    splitName = `${splitName}%`
+  }
+
+  return splitName
+}
+
```

Comment:
I think this should be refactored out into a function file so we can test it independently